src/components/Toolbar.js

```diff
@@ -1,16 +1,114 @@
 import React from "react";
+import Menu from "react-menus";
 
-const toolbarStyle = {color: "white", backgroundColor: "gray", width: "100%", height: 40, position: "fixed"};
+function onClick(events, props, index) {}
+//function onChildClick(event, props, index) {}
+
+const toolbarStyle = {color: "white", backgroundColor: "#C0C0C0", width: "100%", height: 40, position: "fixed"};
+
+/* var itemProps = {};
+ * expander: "Sub Menu"
+ * */
+const items = [
+  {
+    label: "Download this file"
+  },
+  {
+    label: "Documentation"
+  },
+  {
+    label: "TODO: Font"
+  },
+  {
+    label: "Report an Error"
+  },
+  {
+    label: "Discuss Pyret"
+  },
+  {
+    label: "Log Out"
+  }
+];
+
+//TODO
+//so much
+//research google APIs
+class Connect2GDrive extends React.Component {
+  constructor(props){
+    super(props);
+  }
+  render() {
+    return(
+      <button style={{height: "100%", border: "none", fontSize:"15px", fontFamily: "sans-serif", width: 194, backgroundColor: "gray", color: "white"}}>Connect to Google Drive</button>
+    );
+  }
+}
+
+//TODO
+//if clicked this.setState should make running equal to true
+//test and (probably) debug gif img within img
+class RunButton extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {running: true};
+    var normalRunStyle = {height: "100%", border: "none", fontSize:"15px", fontFamily: "sans-serif", width: 150, float: "right", backgroundColor: "#317BCF", color: "white"};
```

Comment:
for all the styling that is the same across different elements, you should create a `styles.js` file that exports appropriate constants.  For example, it might have:

```javascript
export const LIGHT_TEXT_COLOR = 'white';
export const DARK_BACKGROUND_COLOR = 'gray';
export const DEFAULT_BUTTON_WIDTH = 150;
export const TOOLBAR_BUTTON = {
  height: '100%',
  border: 'none',
  fontSize: 15,
  fontFamily: 'sans-serif',
  width: DEFAULT_BUTTON_WIDTH,
  backgroundColor: DARK_BACKGROUND_COLOR,
  color: LIGHT_TEXT_COLOR,
};
```

Then just refer to those constants.

In general when you are writing code, you want to follow the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).